site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -127,4 +127,11 @@
     </span>
 </div>
 <br>
-</div>
\ No newline at end of file
+</div>
+<script>
+    $("input[name='title']").change(function () {
+        if($.trim($(this).val()).length == 0){
+            $(this).val($.trim($(this).val()));
+        }
```

Comment:
Let's get rid of the `if` statement and just use the `$(this).val($.trim($(this).val()));` bit to always trim it. We don't ever want the spaces on the start/end of the string.